### PR TITLE
Fixes Twitter login failures

### LINKED ISSUE #42: Twitter login broken
Twitter popup appears, logs you in and then disappears, but the client remains not logged in.

### PR DESCRIPTION
**Pull request by richsilv**
Also provides error logging for failed login attempts on server, and simplified error logging on client (as Meteor makes it very difficult to provide a non-generic message in this circumstance).

Fixes #42

### CODE REVIEW

**Comment by olizilla** [comment]
Rather than make this part smarter let's simplify things:
- on log in, If there are no eaters, when the first person logs in, we create an eater for them.
- If there are eaters, then you can only log in if there is an eater object with your twitter screen name.

**Comment by richsilv** [comment]
Agreed, but shouldn't it be "if there are no *users*"?

File: server/server.js [hunk 1]
```diff
@@ -146,11 +146,34 @@ Meteor.methods({
   }
 })
 
+var twitterWhitelist = [
+  'richsilvo',
+  'msdosh',
+  '_bmordan',
+  'olizilla',
+  '_alanshaw'
```
Comment (olizilla) [inline server/server.js]: Pls push into settings and update the readme

**Comment by olizilla** [comment]
Ace, thanks.

**Pull request merged**